cookbooks/aws-parallelcluster-platform/resources/fetch_config.rb

```diff
@@ -176,9 +177,10 @@ def write_config_version_file(path)
   end
 
   def wait_cluster_config_file(path)
```

Comment:
Non Blocking. But in later PR's we can use the `write_sync_file` or `wait_sync_file` instead of these different file named wait and write functions